apps/src/templates/instructions/CommitsAndReviewTab.jsx

```diff
@@ -89,6 +90,21 @@ const CommitsAndReviewTab = props => {
     }
   };
 
+  const resolveComment = async (
```

Comment:
nit: maybe `toggleCommentResolved` as this controls both resolving and un-resolving the comment?